pkgs/development/python-modules/pypcap/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, lib, buildPythonPackage, fetchPypi, isPy3k, pythonPackages, pkgs }:
```

Comment:
pass in in individual dependencies, so `dpkt` and `libpcap`, instead of `pythonPackages` and `pkgs`.

Reply:
Thanks for your feedback. I applied the requested changes.